tests/unit/createRouter.ts

```diff
@@ -149,13 +149,14 @@ suite('createRouter', () => {
 	test('dispatch emits navstart event', () => {
 		const router = createRouter();
 
-		let received = '';
+		let received: NavigationStartEvent = null!;
```

Comment:
Okay is that better than casting to `any`? Also would it be good to add a comment with the reasoning?